tests/Tests.FeatureManagement/appsettings.json

```diff
@@ -209,7 +209,7 @@
               "to": 50
             }
           ],
-          "seed": 1234
+          "seed": "1234"
```

Comment:
Why did the seed change from a number to string?

Reply:
Schema dictates it's a string. In .net this happened to work because everything is turned into a string when placed in config.